sdk/tables/azure-data-tables/samples/sample_update_upsert_merge_entities.py

```diff
@@ -109,17 +110,18 @@ def list_all_entities(self):
                 # [START list_entities]
                 # Query the entities in the table
                 entities = list(table.list_entities())
-
+                i = 0
                 for i, entity in enumerate(entities):
                     print("Entity #{}: {}".format(entity, i))
+                    i += 1
```

Comment:
`i` is being set in the for loop on line 114 - why is it being incremented here?

Reply:
You're right, it is not expected to exist.